src/app/pages/components/calendar/calendar.component.ts

```diff
@@ -0,0 +1,145 @@
+import {Component, ElementRef, Input, OnInit, ViewEncapsulation} from '@angular/core';
```

Comment:
I think there should be spaces within the import braces `import { Component, ... }`